packages/web3-eth-accounts/test/fixtures/account.ts

```diff
@@ -392,3 +430,16 @@ export const invalidDecryptData: [[any, string], InvalidKdfError | KeyDerivation
 		new KeyDerivationError(),
 	],
 ];
+
+export const validHashMessageData: [string, string][] = [
+	['🤗', '0x716ce69c5d2d629c168bc02e24a961456bdc5a362d366119305aea73978a0332'],
+	[
+		'Some long text with integers 1233 and special characters and unicode \u1234 as well.',
+		'0xff21294f27c6b1e416215feb0b0b904c552c874c4e11b2314dd3afc1714ed8a8',
+	],
+	[
+		'Some long text with integers 1233 and special characters and unicode \u1234 as well.',
+		'0xff21294f27c6b1e416215feb0b0b904c552c874c4e11b2314dd3afc1714ed8a8',
+	],
+	['non utf8 string', '0x8862c6a425a83c082216090e4f0e03b64106189e93c29b11d0112e77b477cce2'],
+];
```

Comment:
```suggestion
['','0x5f35dce98ba4fba25530a026ed80b2cecdaa31091ba4958b99b52ea1d068adad']
];
```

good to add edge case